CONTRIBUTING.md

```diff
@@ -28,7 +29,7 @@ yarn develop
 ```
 
 This command will start a hugo webserver and serve the `exampleSite`. This works well because the example site uses `hugo-theme-codex`, so any changes
-we make to the codebase can be viewed in real time.
+we make to the codebase can be viewed in real time. Make sure you change `"hugo-theme-codex"` in `exampleSite/config.toml` to the name you gave the theme when you added it. If you followed the README, you should have added the theme using the name `"codex"`. 
```

Comment:
The only issue I have with this instruction is:

> If you followed the README, you should have added the theme using the name `"codex"`.

This is true for users of the theme, but if someone wants to contribute and follows the `CONTRIBUTING` docs, they should be forking / cloning the repo as `hugo-theme-codex`, its default project name.

Reply:
Oh, I just copy-pasted that sentence from the `config.toml` file.

In any case, the sentence stands, as it only talks about the README file and not the CONTRIBUTING one. Maybe we should make the documentation more coherent on that aspect?

Reply:
```suggestion
we make to the codebase can be viewed in real time. 
```

Let's revert this change. I'll then accept this PR to get the YARN docs in, and I'll open another PR to fix the naming scheme for this project.